docs/MachineConfiguration.md

```diff
@@ -115,6 +116,29 @@ Note that for 4.2 clusters this is only supported as a "day 2" operation.
 #### nosmt
 When a machine boots with `nosmt` Kernel Argument, it disables multi-threading on that host and the system will only utilize physical CPU cores. While applying `nosmt` on any node in the cluster, ensure that enough CPU resources are available to schedule all pods, otherwise it can lead to a degraded cluster. For example: a basic 3 master and 3 worker node cluster having 2 physical CPU cores on each node should be fine.
 
+### KernelType
+
+This feature is available with OCP 4.4 and onward releases as both `day 1` and `day 2` operation. It allows to choose between traditional and realtime kernel on an RHCOS node. Supported values are
```

Comment:
You refer to RT as "Real Time", "realtime" and "real-time" throughout this section, should probably make it consistent.

Reply:
It gives a feeling that I am not writing same word again and again ;)
Fixed!